langworld_db_data/adders/listed_value_adder.py

```diff
@@ -4,6 +4,11 @@
 from langworld_db_data.constants.literals import ID_SEPARATOR
 from langworld_db_data.filetools.csv_xls import read_dicts_from_csv, write_csv
 
+FEATURE_ID = "feature_id"
+ID = "id"
+INDEX = "index"
+LINE_NUMBER = "line number"
```

Comment:
please rename to make them more self-explanatory

Reply:
Done!

Reply:
index and line number are fine, but I have doubts about the first two. See https://github.com/Jazyki-Mira/langworld_db_data/pull/244#discussion_r1732635161